start.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
 #
-# Start script for lfp-pay-web
+# Start script for penalty-payment-web
 
 APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 if [[ -z "${MESOS_SLAVE_PID}" ]]; then
     source ~/.chs_env/private_env
     source ~/.chs_env/global_env
-    source ~/.chs_env/lfp-pay-web/env
+    source ~/.chs_env/penalty-payment-web/env
 
-    PORT="${LFP_PAY_WEB_PORT}"
+    PORT="${Penalty_Payment_WEB_PORT}"
```

Comment:
```suggestion
    PORT="${PENALTY_PAYMENT_WEB_PORT}"
```

Reply:
Changed to PENALTY_PAYMENT_WEB_PORT as suggested